### DISCUSSION TITLE
Weird problem with Bottles after updating packages

### DISCUSSION THREAD

**Discussion by Qronikarz** [linuxmint/discussions#104] [Questions]
I have a question since I have no idea where to even star troubleshooting this bug.

Running Mint 21.2. Bottles wasn't updated at all for few months so it stays absolutely the same and it shouldn't be the cause. Few days ago there were some updates. If I had to say the timeframe it must have been before the 3rd of August and somewhere after 29 of July since that was the last time I remember running it without any problems.

Bear in mind, I did not change a single thing in Bottle setting, no different runners or anything like that. The problem is that some apps don't work with normal play button and there is nothing in the terminal logs. When you click on "launch with terminal" button it surprisingly runs, but it has errors in terminal and the performance is horrible with even missing textures. 
![termbutton](https://github.com/linuxmint/discussions/assets/84787215/12e7b80e-01da-4885-8f86-e924d0815b0c)
![menus](https://github.com/linuxmint/discussions/assets/84787215/ce80c265-b491-44a0-a262-f7b3d0fb707e)
Now as you can see there are few problems and I have no idea where to pass them.
1. gtk-critical _gtk_css_corner_value - happens when you click on any menu option. Like for example the 3 dots next to play button in programs list
2. failed to load module xapp - it did happen few times in the past, but was fixed and now it's again here.
3. easyterm VTE warning - that is Bottles terminal, but this error also wasn't present before
4. usr/lib/python3.10 source id was not found - this one probably is the most important since Bottles is a Python app. Maybe something got broken in some updates?

Don't mind the gtk-warning one it's due to & in program name and was already reported.

So question to maybe people that run 21.2 but didn't update anything yet. Do you also have same error when opening menus in Bottles?
I don't want to install it again, but it would be nice to find a cause. I'll also cross-post this topic to Bottles GitHub maybe something can be done from there even though Bottles didn't have any updates.

**Comment by Qronikarz** [comment]
Update:
Mint 21.3 only has xapp and VTE error. I think probably due to some flatpak runtime, but I'll try it once again in Mint 22.

**Reply by Jopp-gh** [reply]
I don't know Bottles, but Lutris, which should be much more robust to run windows apps in general - not just games.

**Reply by Qronikarz** [reply]
I much prefer Bottles. It works perfectly fine, not counting this weird case which happened who knows why. But as I said, it doesn't matter. I'll try on Mint 22 and close this topic either way.

**Comment by Qronikarz** [comment]
Mint 22 beta doesn't have the xapp and VTE errors. Closing as the problem no longer appears.
